### PR TITLE
Adding xtal2png

### PR DESCRIPTION
**Pull request by sgbaird**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sgbaird** [comment]
@conda-forge-admin, please ping conda-forge/help-python

File: recipes/xtal2png/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xtal2png-{{ version }}.tar.gz
+  sha256: a373c7f93610aad5bc54bbd1ef5ac4c730533b43a09750194ee5eeaa93074841
+
+build:
+  number: 0
+  entry_points:
+    - xtal2png = xtal2png.core:run
+  script: {{ PYTHON }} -m pip install . -vv
+
+  noarch: python
```
Comment (BastianZim) [inline recipes/xtal2png/meta.yaml:19]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
  
```

**Comment by BastianZim** [comment]
I'm going to merge this now since it builds but you added an extra line that should be fixed with the next update.

**Comment by sgbaird** [comment]
@BastianZim thanks for approving my first-time contrib to `conda-forge`!

The extra line is related to using [`conda-souschef`](https://github.com/marcelotrevisani/souschef/issues/29) to touch up auto-generated `grayskull` recipes (see e.g. https://github.com/sparks-baird/xtal2png/blob/main/scripts/run_grayskull.py though still integrating into GH actions). Planning to fix before next recipe commit.

https://github.com/pyscaffold/pyscaffold/issues/633

**Comment by BastianZim** [comment]
My pleasure! And welcome to conda-forge. 

Ahh ok makes sense. I think when a new release is done, the bot will probably redo that anyways.

**Comment by sgbaird** [comment]
@BastianZim are you referring to [`regro-cf-autotick-bot`](https://conda-forge.org/docs/maintainer/updating_pkgs.html?highlight=branch#pushing-to-regro-cf-autotick-bot-branch)? If so, this is great to know. I've come across this but either didn't realize what it did or forgot. Thank you for mentioning. In that case, it sounds like I won't need to run `grayskull`, touch it up, and push the new `meta.yaml` to the feedstock for every version update - instead `regro-cf-autotick-bot` will update what it can (other than e.g. dependencies) and then I can touch-up the dependencies manually in the PR per the linked instructions. From there, if a single dependency changes, I only have to do that manual PR editing once, since it will be preserved the next time I release a version? At least that's what I'm getting from the docs, but I'm sure it will be clear once I test it out, too.

**Comment by sgbaird** [comment]
Sounding much more straightforward than a GH actions `grayskull` / `conda-souschef` / [`copy_file_to_another_repo_action`](https://github.com/marketplace/actions/push-a-file-to-another-repository) workflow. Feeling a bit 🤦. But learned a lot struggling through it.

**Comment by BastianZim** [comment]
😄 We all started out that way... And yes, that's the bot I'm talking about. Once a new version is released, it will pick that up and make a PR to the feedstock with the version and SHA updated. You then just need to update dependencies, license and anything else that changed and merge it. And yes, the changes will be preserved since the bot will fork the repo for every update. 
Just make sure that you never push directly to the repo but instead to a fork and then merge that fork if you change something on your own.
You will have push rights to the bot PR so you can either just click on the branch and be brought there or check it out with the CLI or GitHub App.

Having your own workflow only really makes sense if you can automate it fully, meaning including dependencies etc. or if you need a new version on conda-forge immediately as the bot sometimes takes a couple of hours.

**Comment by sgbaird** [comment]
@BastianZim thanks so much! Agreed about only making sense if it can be automated fully or needing new version immediately (hard to imagine a use-case for the latter that isn't related to impatience lol). Really appreciate the clarifications.

**Comment by BastianZim** [comment]
Of course! Feel free to ping me if you have any other questions. :)

**Pull request merged**